src/ray/raylet/worker_pool.cc

```diff
@@ -39,10 +39,6 @@ DEFINE_stats(worker_register_time_ms,
 
 namespace {
 
-// Add this prefix because the worker setup token is just a counter which is easy to
-// duplicate with other ids.
-const std::string kWorkerSetupTokenPrefix = "worker_startup_token:";
-
```

Comment:
Added with
https://github.com/ray-project/ray/pull/20789/files#diff-4c19d6ebd8575189e9439fa5646b567546dc54f64d576a959ce1bb43902d1a03R39

Code using it has since been removed